i18n/es/docusaurus-plugin-content-docs/current/publication-apps/requirements.md

```
@@ -0,0 +1,167 @@
+Es obligatorio proporcionar al equipo de Tiendanube, durante la fase de homologación y junto con la demás documentación compartida, el documento de preguntas frecuentes (FAQ), que contiene las dudas más comunes sobre su aplicación.
+
+Ten especial atención a los contactos de soporte (nivel 1, nivel 2, técnico y comercial) y a la disponibilidad de cuentas de prueba.
+
+Tiendanube ha creado una plantilla para cada categoría que puedes utilizar como guía.
+
+- Las preguntas del documento de preguntas frecuentes (FAQ) variarán según la categoría de su solicitud.
+- Copia la plantilla de documento que aparece a continuación y compártela con su solicitud de aprobación.
+
+> ▶️ **FAQs ERP**  
+> [Template FAQ ERP](https://docs.google.com/document/d/1n0M6LV1FiArb5IPPYI7CXUVFoVPUF7CHR9wuR-3wp94/edit?tab=t.0#heading=h.u4429oqdb513)
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)